windows: Build failing when merging #723

When trying to merge #723 for Windows, a build failure occurs. Here is the relevant build failure snippet:
```
..\src\client\cmd\common_cli.cpp(87): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int
..\src\client\cmd\common_cli.cpp(87): error C2143: syntax error: missing ',' before '*'
..\src\client\cmd\common_cli.cpp(88): error C2244: 'add_instance_names': unable to match function definition to an existing declaration
..\src\client\cmd\common_cli.cpp(87): note: see declaration of 'add_instance_names'
..\src\client\cmd\common_cli.cpp(157): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int
..\src\client\cmd\common_cli.cpp(157): error C2143: syntax error: missing ',' before '*'
..\src\client\cmd\common_cli.cpp(158): error C2244: 'run_cmd': unable to match function definition to an existing declaration
..\src\client\cmd\common_cli.cpp(157): note: see declaration of 'run_cmd'
..\src\client\cmd\common_cli.cpp(173): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int
..\src\client\cmd\common_cli.cpp(173): error C2143: syntax error: missing ',' before '*'
..\src\client\cmd\common_cli.cpp(175): error C2244: 'run_cmd_and_retry': unable to match function definition to an existing declaration
..\src\client\cmd\common_cli.cpp(173): note: see declaration of 'run_cmd_and_retry'
```
It seems for whatever reason, VC does not like the `InstanceNames` type as a return value:(

--- a/src/client/cmd/common_cli.cpp
+++ b/src/client/cmd/common_cli.cpp
@@ -84,7 +84,7 @@ mp::InstanceNames cmd::add_instance_names(const mp::ArgParser* parser)
     return instance_names;
 }
 
-mp::InstanceNames cmd::add_instance_names(const ArgParser* parser, const std::string& default_name)
+mp::InstanceNames cmd::add_instance_names(const mp::ArgParser* parser, const std::string& default_name)
 {
     auto instance_names = add_instance_names(parser);
     if (!instance_names.instance_name_size() && !parser->isSet(all_option_name))
@@ -154,7 +154,8 @@ void check(mp::ParseCode code)
 }
 } // namespace
 
-mp::ReturnCode cmd::run_cmd(const QStringList& args, const ArgParser* parser, std::ostream& cout, std::ostream& cerr)
+mp::ReturnCode cmd::run_cmd(const QStringList& args, const mp::ArgParser* parser, std::ostream& cout,
+                            std::ostream& cerr)
 {
     ArgParser aux_parser{args, parser->getCommands(), cout, cerr};
     check(aux_parser.parse());
@@ -170,7 +171,7 @@ mp::ReturnCode ok2retry(mp::ReturnCode code)
 }
 } // namespace
 
-mp::ReturnCode cmd::run_cmd_and_retry(const QStringList& args, const ArgParser* parser, std::ostream& cout,
+mp::ReturnCode cmd::run_cmd_and_retry(const QStringList& args, const mp::ArgParser* parser, std::ostream& cout,
                                       std::ostream& cerr)
 {
     return ok2retry(run_cmd(args, parser, cout, cerr));

